doc/sphinx/source/remote.rst

```diff
@@ -0,0 +1,441 @@
+=============
+Remote Driver
+=============
+
+Background
+----------
+
+PoCL-Remote is an OpenCL driver which forwards OpenCL commands to
+a remote server over network. The remote OpenCL devices are listed in
+the local OpenCL platform device list and each each remote device can
+be used like it was a local OpenCL device. The properties of a remote
+device as queried via clGetDeviceInfo() mirror the remote physical
+device's properties etc.
+
+The key differentiating aim of PoCL-Remote to other similar remote
+forwarding OpenCL implementations is that the server side component
+is capable of autonomously scheduling commands based on event dependencies
+as well as communicating in peer-to-peer fashion with other servers in order
+to not tax the client's network connection with signaling and buffer
+migrations between servers. The overall design focuses also on execution
+latency, not only throughput, and is still targeted to also support
+interactive low latency applications such as mixed reality.
+
+One the client side PoCL-Remote is a pocl driver backend that controls
+remote devices via a control protocol and transparently exposes them to
+applications as if they were local devices. On the server side, there is the
```

Comment:
Perhaps advertise the aspect that we can build a Nano-PoCL with low requirements on the target system and compiler, and point to the nanodrone case. This is why the client-side driver and the core PoCL is written in simple C instead of C++, for example.